test_conformance/images/kernel_read_write/test_iterations.cpp

```diff
@@ -85,7 +85,7 @@ static const char *lodOffsetSource =
 static const char *offsetSource =
 "   int offset = tidY*get_image_width(input) + tidX;\n";
 
-#define ABS_ERROR( result, expected ) ( fabsf( (float)expected - (float)result ) )
+#define ABS_ERROR(result, expected) (fabs(expected - result))
```

Comment:
I was thinking of something higher level but that goes beyond the scope of this PR. Fine with the macro for now.